3rdparty/re2/re2/regexp.cc

```diff
@@ -23,7 +23,7 @@
 
 namespace re2 {
 
-// Constructor.  Allocates vectors as appropriate for operator.
+// Constructor.  Allocates _vectors as appropriate for operator.
```

Comment:
Is a real motivation to rename it from vectors to _vectors or is it juser unfortunate replace all by IDE.

Reply:
shit. Of course not, this is nuts. I am learning to use clion, it does somewhat crazy things.